src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+
+@WebServlet("/login-handler")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Set<String> TAs = new HashSet<String>();
+    TAs.addAll(Arrays.asList(
+        new String[] {"TA1", "TA2"}));
```

Comment:
Nit: These are supposed to be emails, so I recommend more realistic examples, e.g. jbiden@email.com or hchu@sps.com

Reply:
Updated in newest commit